content/src/content/jcr_root/apps/core/wcm/sandbox/commons/ui/clientlibs/panelcontainer/js/panelContainerUtils.js

```diff
@@ -0,0 +1,141 @@
+(function(ns) {
+    "use strict";
+
+    /**
+     * Panel Container Utilities
+     *
+     * @namespace
+     * @alias CQ.CoreComponents.panelcontainer.utils
+     * @type {{}}
+     */
+    CQ.CoreComponents.panelcontainer.utils = {
+
+        /**
+         * Checks whether an [Editable]{@link Granite.author.Editable} is a Panel Container
+         *
+         * @param {Granite.author.Editable} editable The [Editable]{@link Granite.author.Editable} to check
+         * @returns {Boolean} True if the Editable is a Panel Container, false otherwise
+         */
+        isPanelContainer: function(editable) {
+            var panelContainerType = getPanelContainerType(editable);
+            return (panelContainerType !== undefined);
+        },
+
+        /**
+         * Returns the Panel Container definition associated with an [Editable]{@link Granite.author.Editable}
+         *
+         * @param {Granite.author.Editable} editable The Panel Container [Editable]{@link Granite.author.Editable}
+         * @returns {Object} The Panel Container Type definition, undefined if none is associated
+         */
+        getPanelContainerType: function(editable) {
+            return getPanelContainerType(editable);
+        },
+
+        /**
+         * Returns the Panel Container HTML element associated with an [Editable]{@link Granite.author.Editable}
+         *
+         * @param {Granite.author.Editable} editable The Panel Container [Editable]{@link Granite.author.Editable}
+         * @returns {HTMLElement} The HTML Element match for the Panel Container, undefined if none is associated
+         */
+        getPanelContainerHTMLElement: function(editable) {
+            var container = getPanelContainerType(editable);
+            var element;
+
+            if (container) {
+                if (editable.dom[0].matches(container.selector)) {
+                    element = editable.dom[0];
+                } else {
+                    var children = editable.dom[0].children;
+                    for (var i = 0; i < children.length; i++) {
+                        var child = children[0];
+                        if (child.matches(container.selector)) {
+                            element = child;
+                            break;
+                        }
+                    }
+                }
+            }
+
+            return element;
+        },
+
+        /**
+         * Returns Panel Container [Editable]{@link Granite.author.Editable}'s child items
+         *
+         * @param {Granite.author.Editable} editable The Panel Container {@link Granite.author.Editable}
+         * @returns {Array<Granite.author.editable>} The Panel Container child editables
+         */
+        getPanelContainerItems: function(editable) {
+            var children = [];
+            if (editable.isContainer()) {
+                children = editable.getChildren().filter(isDisplayable);
+            }
+            return children;
+        }
+    };
```

Comment:
This method returns the editable's children. Shouldn't it return the Panel Container child editables by first calling:
`var container = getPanelContainerType(editable);`